src/theme/fw.ts

```diff
@@ -0,0 +1,112 @@
+import type { Theme } from './theme'
+
+const fontFamily =
+  'Avenir Next, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif'
+
+export const colors = {
+  primaryGradient: 'linear-gradient(254deg, #f78361, #f54b64)',
+  primary: '#ff7061',
+  blk1: '#121212',
+  gray1: '#171717',
+  gray2: '#2b2b2b',
+  gray3: '#404040',
+  gray4: '#9b9b9b',
+  gray5: '#bdbdbd'
+}
+
+const fontSizes = {
+  xxsmall: '10px',
+  xsmall: '12px',
+  small: '14px',
+  medium: '16px',
+  large: '18px',
+  xlarge: '20px',
+  xxlarge: '24px',
+  xxxlarge: '30px'
+}
+
+const fontOffsets = {
+  xxsmall: ['1px', '3px'],
+  xsmall: ['1px', '4px'],
+  small: ['2px', '4px'],
+  medium: ['2px', '5px'],
+  large: ['3px', '6px'],
+  xlarge: ['4px', '6px'],
+  xxlarge: ['5px', '7px'],
+  xxxlarge: ['6px', '8px']
+}
+
+const fwTheme: Theme = {
+  colors: colors,
+  fontSizes: fontSizes,
+  fontOffsets: fontOffsets,
+  button: {
+    bg: 'transparent',
+    borderRadius: '500px',
+    color: 'white',
```

Comment:
Shouldn't we use a standardized white/black defined in `colors`? Thinking about the case if we choose to use a different `white` for the standard white later.